webpack.config.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path')
+const enter = path.join(__dirname, './client/index.jsx')
+const out = path.join(__dirname, 'public')
+module.export = {
+  entry: enter,
+  output: {
+    path: out,
+    filename: 'bundle.js'
+  },
+  module: {
+    rules: [{
+      test: /\.jsx$/,
+      use:{
+        loader: 'babel-loader',
+        options: {
```

Comment:
The spacing looks off here? Can't tell if that's Github rendering it oddly. Are we using a linter?